src/main/kotlin/apply/application/github/GithubDtos.kt

```diff
@@ -0,0 +1,20 @@
+package apply.application.github
+
+import com.fasterxml.jackson.core.JsonParser
+import com.fasterxml.jackson.databind.DeserializationContext
+import com.fasterxml.jackson.databind.JsonDeserializer
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize
+
+private class CommitDeserializer : JsonDeserializer<CommitResponse>() {
+    override fun deserialize(p: JsonParser, ctxt: DeserializationContext): CommitResponse {
```

Comment:
```suggestion
    override fun deserialize(parser: JsonParser, deserializationContext: DeserializationContext): CommitResponse {
```
c : 길더라도 변수명을 확실하게 지어주는게 좋을 것 같아요 🧚‍♀️